versions/2016-10-31.md

```diff
@@ -319,6 +321,7 @@ Property Name | Type | Description
 Stream | `string` | **Required.** ARN of the Amazon Kinesis stream.
 StartingPosition | `string` | **Required.** One of `TRIM_HORIZON` or `LATEST`.
 BatchSize | `integer` | Maximum number of stream records to process per function invocation.
+Enabled | `bool` | Indicates whether Lambda begins polling the event source.
```

Comment:
This line was added, the rest are style changes

Reply:
This isn't explicitly defined in the translator https://github.com/awslabs/serverless-application-model/blob/develop/samtranslator/model/eventsources/pull.py#L51. I may be wrong, but I don't think it'll copy over any additional values you define unless they're explicitly defined in the translator.